File: CS/Dashboard_AggrSalesByMonths/Form1.cs
using DevExpress.XtraBars.Ribbon;

namespace Dashboard_AggrSalesByMonths {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"..\..\Data\SalesByMonths.xml");
        }
    }
}

File: VB/Dashboard_AggrSalesByMonths/Form1.vb
Imports DevExpress.XtraBars.Ribbon

Namespace Dashboard_AggrSalesByMonths

    Public Partial Class Form1
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            dashboardDesigner1.CreateRibbon()
            dashboardDesigner1.LoadDashboard("..\..\Data\SalesByMonths.xml")
        End Sub
    End Class
End Namespace
